zend_abstract_interface/config/php7-8/config_runtime.c

```diff
@@ -20,12 +20,48 @@ void zai_config_replace_runtime_config(zai_config_id id, zval *value) {
     ZVAL_COPY(rt_value, value);
 }
 
+static inline zval *zai_config_runtime_config_value(zai_config_memoized_entry *memoized, zval *updated) {
+    ZAI_ENV_BUFFER_INIT(buf, ZAI_ENV_MAX_BUFSIZ);
+
+    int16_t name_index = 0;
+    for (; name_index < memoized->names_count; name_index++) {
```

Comment:
name_index is not used after the loop, could you put it into the for stmt?

Reply:
I must have been searching the wrong file, I found a use outside the loop the first time I looked 😕 

moving it ...